klippy/extras/print_stats.py

```diff
@@ -59,13 +64,30 @@ def _note_finish(self, state, error_message = ""):
             self.init_duration = self.total_duration - \
                 self.prev_pause_duration
         self.print_start_time = None
+    cmd_SET_PRINT_STATS_INFO_help = "Pass slicer info like layer act and " \
```

Comment:
This is in response to Kevin's last comment about `SET_PRINT_STATS_INFO` without `CURRENT_LAYER` clearing the current layer. I thought something like this would be more intuitive.

```
    def cmd_SET_PRINT_STATS_INFO(self, gcmd):
        total_layer = gcmd.get_int("TOTAL_LAYER", self.info_total_layer, \
                                   minval=0)
        current_layer = gcmd.get_int("CURRENT_LAYER", self.info_current_layer, \
                                     minval=0)
        if total_layer == 0:
            self.info_total_layer = None
            self.info_current_layer = None
        elif total_layer != self.info_total_layer:
            self.info_total_layer = total_layer
            self.info_current_layer = 0
        if current_layer != self.info_current_layer
            if current_layer < total_layer:
                self.info_current_layer = current_layer
            else:
                self.info_current_layer = total_layer
```

This way the current layer gets reset only when the total layer value changes (which also could be important if this command grows more functionality later).